CopenhagenCityArchives/kbh-billeder
# Mode byrummnet only has wms maps

This is the finalizing PR on making the new maps work in the mode byrummet. 

![image](https://github.com/CopenhagenCityArchives/kbh-billeder/assets/17847632/cec8ad84-b61c-45a1-b0da-7f99047cda9c)


# Changes
We can now select between the 4 (test) maps we have from GEO. 
All functionality should still be there. The maps should only be available to Byrummet for now. 

## Known Issues
The load and render of the WMS pictures are really slow (GEO server fault - it takes upwards of 16 seconds to deliver). Because of the TimeWarp implementation, the pink circle doesn't show up until the map is ready. That leaves the user without any really good indicator of something happening. 
I don't have a fix on my hands. 
![image](https://github.com/CopenhagenCityArchives/kbh-billeder/assets/17847632/32f3b9c7-6fe0-495c-83d3-0d7ec96e5e6f)


## How to test
For every map, test that the map can be interacted with. Assets chosen, mouse used to drag, zoom in and out. Use the arrow keys to make sure that they can still be used to move the map around. 
Go to a single assets page, go the map, switch between the maps and ensure that no new maps have become available. 
Go to map search and check for the same thing. 
Now go to **Byrummet**. There should be no more than 4 maps to chose from.
